spec/tic_tac_toe_spec.rb

```diff
@@ -0,0 +1,65 @@
+require_relative '../lib/player'
+require_relative '../lib/game'
+require_relative '../lib/board'
+require_relative '../lib/modules/helper'
+
+describe Game do
+  let(:player_one) { Player.new('Fred', 'x') }
+  let(:player_two) { Player.new('Alex', 'o') }
+  let(:game) { Game.new(player_one, player_two) }
+  let(:game_class) { Game }
+
+  describe '#choose_signs' do
+    it 'returns an array with the random signs' do
+      expect(game_class.choose_signs).to be_an(Array)
+    end
+  end
+
+  describe '#validate_name' do
+    it 'returns false only when name is short' do
+      expect(game_class.validate_name('Alan')).to_not be_falsy
+    end
+  end
+
+  describe '#validate_move' do
+    it 'return -1 when the move is already taken' do
+      expect(game.validate_move('2')).not_to eql(-1)
+    end
+
+    it 'return -2 when is an invalid move' do
+      expect(game.validate_move('10')).to eql(-2)
+    end
+
+    it 'return the valid number' do
+      expect(game.validate_move('5')).to eql('5')
+    end
+  end
+  describe '#getcurrent_player' do
+    it 'return an array  with two player object' do
+      expect(game.getcurrent_player).to be_an(Array)
+    end
+  end
+
+  describe '#winner?' do
+    it 'return an string' do
+      expect(game.winner?).to_not be_an(String)
+    end
+  end
+end
```

Comment:
- [ ] Please add positive scenario tests.

**IMPORTANT!** Please go through all your tests and missing tests i.e., add negative scenario tests (`.not_to eql`) where negative tests are missing and positive scenario tests (`.to eql`) where positive tests are missing.